filament/src/Renderer.cpp

```diff
@@ -728,11 +728,13 @@ FrameGraphId<FrameGraphTexture> FRenderer::refractionPass(FrameGraph& fg,
             h = config.vp.height * config.scale.x;
         }
 
+        const TextureFormat format = (view.getBlendMode() == BlendMode::TRANSLUCENT) ? mHdrTranslucent : mHdrQualityMedium;
+        
         input = ppm.opaqueBlit(fg, input, {
                 .width = w,
                 .height = h,
                 .levels = roughnessLodCount,
-                .format = TextureFormat::R11F_G11F_B10F,
+                .format = format,
```

Comment:
This is the only thing I'm sad about (doubling the memory if we have to use RGBA16F) and would be curious whether this increases our chances of crashing on lower end devices. I think it is worth a mention in the what-to-test part to see if we made things more problematic on the low end or not.

Reply:
Fair point, I'll add a note on the app side PR. Do you think we should consider `RGB10_A2` for when a lower render quality is set on the `View`? It'll clip HDR data, and reduce alpha precision badly, but it seems like an acceptable compromise for content inside a refractive object.

Reply:
Discussed this in person: let's roll the way we are right now and ask QA to test the stability of this version VS the previous in terms on lower end devices. If it's the same, we are golden! RGB10_A2 is too dicey.